DOC-1321: RS CLI Utilities - rladmin suffix (#1917)
RedisLabs/redislabs-docs: content/rs/references/cli-utilities/rladmin/suffix.md

alwaysopen: false
toc: "true"
headerRange: "[1-2]"
categories: ["RS"]
aliases:
---

`rladmin suffix` manages the DNS suffixes in the cluster.

## `suffix add`

Adds a DNS suffix to the cluster.

``` sh
rladmin suffix add name <name>
                    [default]
                    [internal]
                    [mdns]
                    [use_aaaa_ns]
                    [slaves <ip>..]
```

### Parameters

| Parameter | Type/Value       | Description                                                                                   |
|-----------|------------------|-----------------------------------------------------------------------------------------------|
| name      | string           | DNS suffix to add to the cluster                                                             |
| default   |                  | Sets the given suffix as default. If there is already a default set, this will overwrite it.  |
| internal  |                  | Forces the given suffix to use private IPs                                                    |
| mdns      |                  | Activates multicast DNS support on the given suffix                                           |

> | slaves    | one or more frontend DNS server IPv4 addresses | The given suffix will notify the frontend DNS servers when a change in frontend DNS has occured |

Reviewer: ```suggestion
| slaves    | list of IPv4 addresses | The given suffix will notify the frontend DNS servers when a change in frontend DNS has occurred |
```
